Calculator.MR1R0B0T/CalculatorApp/CalculatorApp/Program.cs

```diff
@@ -4,7 +4,116 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+            Console.WriteLine("Console Calculator Game");
+            Console.WriteLine("-----------------------");
+            Console.WriteLine("Please enter a key to continue. . .");
+            Console.ReadLine();
+            bool showMenu = true;
+            while (showMenu)
+            {
+                Menu();
+            }
+        }
+
+        public static void Menu()
+        {
+            Console.Clear();
+            Console.WriteLine("\t\t\t\t\tHere are the menus for the game: ");
+            string menu = $"\t\t\t\t\t\t 1 - Addition \n \t\t\t\t\t\t 2 - Subtraction \n \t\t\t\t\t\t 3 - Multiplication \n \t\t\t\t\t\t 4 - Division \n\n\n";
+            Console.WriteLine(menu);
+            Choice();
+        }
+
+        public static void Choice()
+        {
+            string choiceMessage = "Please input the type of game you want to play: ";
+            Console.Write(choiceMessage);
+            var choices = Console.ReadLine();
+
+            switch (choices)
+            {
+                case "1":
+                    Console.Clear();
+                    AdditionGame();
+                    break;
+                case "2":
+                    Console.Clear();
+                    SubtractionGame();
+                    break;
+                case "3":
+                    Console.Clear();
+                    MultiplicationGame();
+                    break;
+                case "4":
+                    Console.Clear();
+                    DivisionGame();
+                    break;
+                default:
+                    Console.Clear();
+                    Console.WriteLine("Out of bounds. Please try again.");
+                    Environment.Exit(0);
+                    break;
+            }
+        }
+
+        //Addition
+        public static void AdditionGame()
+        {
+            Console.WriteLine("\t\t\t\t\t\tWelcome to the Addition Game\n\n\n");
+            Console.WriteLine("Please input the 2 numbers: ");
+            Console.Write("First Number: ");
+            int num1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Second Number: ");
+            int num2 = Convert.ToInt32(Console.ReadLine());
```

Comment:
This does not handle validating where the user has entered two numbers that can be parsed. Currently, there is an unhandled exception if the user types in an invalid input. You should consider creating a separate method that is responsible for validating that the user's input is indeed a number. If they do not enter a number, the program should re-prompt them for a valid input.